CONTRIBUTING.md

```diff
@@ -0,0 +1,52 @@
+# Contribution Guide
+
+## Getting Started
+
+### Prerequisites
+Please make sure you have the following programs installed:
+- [Python 3.9](https://www.python.org/downloads/)
+- [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+- [Virtual Box](https://www.virtualbox.org/wiki/Downloads)
+- [Vagrant](https://www.vagrantup.com/downloads)
+
+IMPORTANT:
+
+- Note that Vagrant may not work on some machines (such as Apple M1-based machines), so please make sure that you can find a suitable solution.
+
+### Setting up
+1. Fork the repository.
+2. Clone the repository locally.
+3. Create a new branch.
+4. Open any terminal and navigate to the project directory.
+5. Run the `vagrant up` command.
+
+After the dependencies were installed and the VM initialized, you can proceed working on the project code.
+
+## Development Process
```

Comment:
I've forgot adding these comments. Right now I've uploaded a new PR, containing these recommendations. Thank you.

Reply:
It seems that you have responded to all of the reviews except for the one suggesting that pull requests should be made as small as possible. While you did mention in your response that the description of the PR should be concise, it may also be helpful to consider the amount of files and code logic in the PR to make it as minimal as possible

Reply:
Thank you for pointing that out. I really missed that part. It's now fixed and an updated version should be up soon.